indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -499,6 +499,7 @@
           ;; associated services
           :service-names (m/doc-values m/string-field-mapping)
           :service-names-lowercase (m/doc-values m/string-field-mapping)
+          :service-types-lowercase (m/doc-values m/string-field-mapping)
```

Comment:
`doc-value` is on by default, no need to explicitly set it. Too bad we didn't clean up the other fields in services mapping while we did the ES upgrade.

Reply:
I am not sure if you are asking me to change it or are just informing me that the definition is not needed?  I didn't know and I was just following how the code is written to be consistent.  Should I go through this class and clean it up?  Although I don't know exactly what can be removed, but I can look it up.

Reply:
I think you can just leave it the way it is to be consistent with the other existing fields. But when adding new fields in collection or granule mappings in the future, you don't need to add the `doc-values`.